### PR TITLE
Clickhouse Approx/Distinct Counts

### PR DESCRIPTION
**Pull request by vigneshshettyin**
User using this package will be able to use Clickhouse functions such has uniq, uniqExact, uniqTheta, uniqCombined, UniqCombined64 & uniqHLL12 to get Approx/Exact counts, this will be a optimized alternative for COUNT(DISTINCT var)

Ref : https://clickhouse.com/docs/en/sql-reference/aggregate-functions/reference/uniq

### CODE REVIEW

**Comment by jayvynl** [comment]
Thanks. Could you add some basic tests for these functions？

**Comment by vigneshshettyin** [comment]
> Thanks. Could you add some basic tests for these functions？

Welcome. I have updated my PR with required basic test cases.

**Review by jayvynl — APPROVED** [review]

File: clickhouse_backend/models/__init__.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from .functions import __all__ as fucntions_all
 from .indexes import *  # noqa: F401,F403
 from .indexes import __all__ as indexes_all
+from .count import * # noqa: F401,F403
```
Comment (jayvynl) [inline clickhouse_backend/models/__init__.py:11]: add `from .aggregates import __all__ as aggregates_all`

File: clickhouse_backend/models/count.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
+from django.db.models import Aggregate
+from django.db.models.expressions import Star
+from django.db.models.fields import IntegerField
+
```
Comment (jayvynl) [inline clickhouse_backend/models/count.py:4]: 1. Maybe renamed to `aggregates.py` following django code structure.
2. Add __all__ variable.

File: tests/count/tests.py [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+            },
+            {
+                "date": "2022-05-23",
+                "user": "alice",
+                "show": "Game of Thrones",
+                "episode": "S1E1",
+            },
+        ]
+
+        for item in data_list:
```
Comment (jayvynl) [inline tests/count/tests.py:142]: I think a single bulk_create is better.

File: clickhouse_backend/models/__init__.py [hunk 4]
```diff
@@ -7,11 +7,15 @@
 from .functions import __all__ as fucntions_all
 from .indexes import *  # noqa: F401,F403
 from .indexes import __all__ as indexes_all
+from .count import *  # noqa: F401,F403
```
Comment (jayvynl) [inline clickhouse_backend/models/__init__.py:10]: this should be `from .aggregates import *`

**Comment by vigneshshettyin** [comment]
Hi, @jayvynl can you please re-start the build job?

**Comment by jayvynl** [comment]
> Hi, @jayvynl can you please re-start the build job?

Sure!

I think I'd better add a contributing guide.

**Comment by vigneshshettyin** [comment]
> > Hi, @jayvynl can you please re-start the build job?
> 
> Sure!
> 
> I think I'd better add a contributing guide.

Yes, that will be great! @jayvynl

**Comment by vigneshshettyin** [comment]
Can you merge the PR? 
@jayvynl

**Comment by vigneshshettyin** [comment]
Hey, observed that coverage dropped by 0.2% in last build. In commit (https://github.com/jayvynl/django-clickhouse-backend/pull/66/commits/d99dadf6eb3c4194d84c33609dfa3a8fdf442f4e) I have improved AggregatesTestCase. I request you to please re-start the build @jayvynl

**Review by jayvynl — APPROVED** [review]

**Comment by jayvynl** [comment]
> Hey, observed that coverage dropped by 0.2% in last build. In commit (https://github.com/jayvynl/django-clickhouse-backend/pull/66/commits/d99dadf6eb3c4194d84c33609dfa3a8fdf442f4e) I have improved AggregatesTestCase. I request you to please re-start the build @jayvynl 

Great.

**Pull request merged**